.github/workflows/nv-a6000.yml

```diff
@@ -47,6 +47,7 @@ jobs:
       - name: Install deepspeed
         run: |
           python -m pip install docutils==0.18.1 jinja2==3.0 urllib3==1.26.11 ninja
+          python -m pip install thinc spacy confection --upgrade
```

Comment:
Previous versions of these packages didn't support pydantic > 2, so this works around the issue for now, ideally updating them in the container image in another PR.

Reply:
@mrwyattii - is it worth adding a comment here for why these were upgraded?

Reply:
Added.